docs/gpu-support.md

```diff
@@ -0,0 +1,140 @@
+
+This automation currently only supports machine type `a2`. Machine type `n1` can
+also have guest accelerators attached, however the type and count
+cannot be determined automatically like with `a2`.
+
+[vm-instance]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/modules/compute/vm-instance
+[node-group]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/modules/compute/schedmd-slurm-gcp-v5-node-group
+[controller]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/modules/scheduler/schedmd-slurm-gcp-v5-controller
+[login]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/modules/scheduler/schedmd-slurm-gcp-v5-login
+[omnia-install]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/modules/scripts/omnia-install
+[htcondor-install]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/modules/scripts/htcondor-install
+[htcondor-configure]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/modules/scheduler/htcondor-configure
+[htcondor-execute-point]: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/modules/compute/htcondor-execute-point
+
+## Troubleshooting and tips
+
+* To list accelerator types and availability by region, run
+  `gcloud compute accelerator-types list`. The information is also available in
+  the Google Cloud documentation [here](https://cloud.google.com/compute/docs/gpus/gpu-regions-zones).
+* Deployment time of VMs with many guest accelerators can take longer.
+
+### Slurm on GCP
+
+When deploying a Slurm cluster with GPUs, we highly recommend using the
+modules based on Slurm on GCP version 5 (`schedmd-slurm-gcp-v5-*`). The
+interface is more consistent with HPC Toolkit standards and more functionality
+is available to support, debug and workaround any issues related to GPU
+resources.
+
+#### Interface Considerations
+
+The Slurm on GCP v4 modules (`SchedMD-slurm-on-gcp-*`) have a different
+interface for defining attached accelerators, `gpu_type` and `gpu_count`. These
+must be set even if the machine type implies GPUs.
+
+The Slurm on GCP v5 HPC Toolkit modules (`schedmd-slurm-gcp-v5-*`) have two
+variables that can be used to define attached GPUs. The variable
+`guest_accelerators` is the recommended option as it is consistent with other
+modules in the HPC Toolkit. The setting `gpus` can be set as well, which
+provides consistency with the underlying terraform modules from the
+[Slurm on GCP repo][slurm-gcp].
+
+#### Timeouts when deploying a compute VM
+
+As mentioned above, VMs with many guest accelerators can take longer to deploy.
+Slurm sets timeouts for creating VMs, and it's possible for high GPU
+configurations to push past the default timeout. The timeout in the Slurm on GCP
+v4 HPC Toolkit modules (`SchedMD-slurm-on-gcp-*`) due to a timeout that cannot
+be increased, therefore we recommend using the Slurm on GCP v5 modules.
```

Comment:
I think this phrase is fragmented. You probably mean:
```
The timeout in the Slurm on GCP
v4 HPC Toolkit modules (`SchedMD-slurm-on-gcp-*`) cannot
be increased and, therefore, (...)
```

Reply:
Done